src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/CreateNewImageTests.cs

```diff
@@ -133,7 +133,7 @@ public void ParseContainerProperties_EndToEnd()
     /// <summary>
     /// Creates a console app that outputs the environment variable added to the image.
     /// </summary>
-    [DockerAvailableFact]
+    [DockerAvailableFact(Skip = "https://github.com/dotnet/sdk/issues/36160")]
```

Comment:
if you'd kept the test targeting CurrentTargetFramework, would it have needed to be skipped?

Reply:
I tried to change the test to CurrentTargetFramework and got an error message. Like this:

>  StdErr:
> Error: Invalid option(s):
> -f net8.0
> 'net8.0' is not a valid value for -f. The possible values are:
> net9.0   - Target net9.0